libs/build.gradle.kts

```diff
@@ -0,0 +1,6 @@
+plugins {
+    id("java")
+}
+
+group = "org.qpay"
+version = "unspecified"
```

Comment:
?

Reply:
it's automatically generated, so i thought it needs to determine address of module

Reply:
Get it back and set version